html-aam/META.yml

```diff
@@ -53,3 +53,8 @@ links:
           subtest: el-span
         - test: roles.html
           subtest: el-u
+    - product: safari
+      url: ' '
```

Comment:
@KyleJu Any idea what to do for this empty URL?

I rebased this locally and I get:

```
--- FAIL: TestParseMetadata (0.35s)
    --- FAIL: TestParseMetadata/../html-aam/META.yml (0.00s)
        parse_test.go:58: 
                Error Trace:    parse_test.go:58
                Error:          Expected nil, but got: &url.Error{Op:"parse", URL:" ", Err:(*errors.errorString)(0xc0001a6110)}
                Test:           TestParseMetadata/../html-aam/META.yml
```

<details>
  <summary>Conflict resolved html-aam/META.yml</summary>
  
  ### Code
  ```yaml
links:
    - product: chrome
      url: https://github.com/w3c/html-aam/issues/474
      results:
        - test: table-roles.html
          subtest: el-thead
        - test: table-roles.html
          subtest: el-tbody
        - test: table-roles.html
          subtest: el-tfoot
    - product: edge
      url: https://github.com/w3c/html-aam/issues/474
      results:
        - test: table-roles.html
          subtest: el-thead
        - test: table-roles.html
          subtest: el-tbody
        - test: table-roles.html
          subtest: el-tfoot
    - product: safari
      url: https://github.com/w3c/html-aam/issues/474
      results:
        - test: table-roles.html
          subtest: el-thead
        - test: table-roles.html
          subtest: el-tbody
        - test: table-roles.html
          subtest: el-tfoot
    - product: safari
      url: https://bugs.webkit.org/show_bug.cgi?id=253751
      results:
        - test: roles.html
          subtest: el-a-no-href
        - test: roles.html
          subtest: el-address
        - test: roles.html
          subtest: el-bdi
        - test: roles.html
          subtest: el-bdo
        - test: roles.html
          subtest: el-data
        - test: roles.html
          subtest: el-i
        - test: roles.html
          subtest: el-pre
        - test: roles.html
          subtest: el-q
        - test: roles.html
          subtest: el-samp
        - test: roles.html
          subtest: el-small
        - test: roles.html
          subtest: el-span
        - test: roles.html
          subtest: el-u
    - product: safari
      url: https://bugs.webkit.org/show_bug.cgi?id=203257
      results:
        - test: roles.html
          subtest: el-strong
        - test: roles.html
          subtest: el-em
        - test: roles.html
          subtest: el-b
    - product: safari
      url: ' '
      results:
        - test: roles.html
          subtest: el-u

  ```
</details>

Reply:
PR should be closed!